UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Logging/EventIds.cs

```diff
@@ -79,7 +79,27 @@ public enum EventIds
         /// <summary>
         /// 910018 -  Unhandled Exception Occured.
         /// </summary>
-        UnhandledCleanUpException = 910019
+        UnhandledCleanUpException = 910019,
+        /// <summary>
+        /// 910006 -  Maritime safety information create new RNW record request started.
+        /// </summary>
+        MSICreateNewRNWRecordStart = 910006,
```

Comment:
Please correct the event id's.  Event id from 910006 to 910010 already used. and next should start from 910020.
```suggestion
        MSICreateNewRNWRecordStart = 910020,
```

Reply:
done required changes